### ISSUE TITLE
Crashing while running on linux xlcore

### ISSUE THREAD

**Issue by braccae** [fitzchivalrik/gentletouch#6]
I cannot confirm right now if this happens while playing on Windows, but while playing on linux using xlcore, the game will randomly crash while in an instanced city. [Linked](https://gist.github.com/Braccae/9ddbff2f274afa45551e861088cea8ac) is a gist of my dalamud log. The second exception in the log might not have to do with this plugin specifically so I've also made them aware as well over in goat place.

When I have access to a Windows machine, I'll check it there as well. This is a wonderful plugin, especially the vibration when you're near an aetheryte current and I'd like to be able to use it again!

**Comment by fitzchivalrik** [comment]
Thanks for reporting.

You say randomly, so in general, the normal game vibrations and the plugin are working, but sometimes in cities it crashes? 'Instanced City' meaning a city having an instance (because of too many people)?

**Comment by braccae** [comment]
Yes, I'm not sure if it's because of too many people however whenever I am in an instanced city it's usually limsa so that may very well be the cause. Normal vibrations work in combat and with aether currents. I haven't had the opportunity to access windows still so I apologize that I still haven't tested it there yet. (I rely on docker heavily for work and WSL is a pain that's broken my windows install more than once)

**Comment by fitzchivalrik** [comment]
The crashing stops when you remove GentleTouch, right? 
Currently, I am, unfortunately, unsure why it would crash in that situation you describe but not in others, but when you remove GentleTouch and experience no crashes at all then it must be because of GentleTouch/some kinda interaction which happens with GentleTouch installed.

Sorry for the inconvenience!
